2023-02-15 00:11:00,433 INFO    StreamThr :3648 [internal.py:wandb_internal():90] W&B internal server running at pid: 3648, started at: 2023-02-15 00:11:00.432245
2023-02-15 00:11:00,435 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: status
2023-02-15 00:11:00,443 INFO    WriterThread:3648 [datastore.py:open_for_write():85] open: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/run-untbe0sx.wandb
2023-02-15 00:11:00,444 DEBUG   SenderThread:3648 [sender.py:send():334] send: header
2023-02-15 00:11:00,460 DEBUG   SenderThread:3648 [sender.py:send():334] send: run
2023-02-15 00:11:01,067 INFO    SenderThread:3648 [dir_watcher.py:__init__():216] watching files in: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/files
2023-02-15 00:11:01,067 INFO    SenderThread:3648 [sender.py:_start_run_threads():1068] run started: untbe0sx with start time 1676387460.436241
2023-02-15 00:11:01,067 DEBUG   SenderThread:3648 [sender.py:send_request():361] send_request: summary_record
2023-02-15 00:11:01,067 INFO    SenderThread:3648 [sender.py:_save_file():1319] saving file wandb-summary.json with policy end
2023-02-15 00:11:01,074 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: check_version
2023-02-15 00:11:01,075 DEBUG   SenderThread:3648 [sender.py:send_request():361] send_request: check_version
2023-02-15 00:11:01,145 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: run_start
2023-02-15 00:11:01,156 DEBUG   HandlerThread:3648 [system_info.py:__init__():31] System info init
2023-02-15 00:11:01,156 DEBUG   HandlerThread:3648 [system_info.py:__init__():46] System info init done
2023-02-15 00:11:01,156 INFO    HandlerThread:3648 [system_monitor.py:start():151] Starting system monitor
2023-02-15 00:11:01,156 INFO    SystemMonitor:3648 [system_monitor.py:_start():116] Starting system asset monitoring threads
2023-02-15 00:11:01,156 INFO    HandlerThread:3648 [system_monitor.py:probe():172] Collecting system info
2023-02-15 00:11:01,157 INFO    SystemMonitor:3648 [interfaces.py:start():168] Started cpu
2023-02-15 00:11:01,157 INFO    SystemMonitor:3648 [interfaces.py:start():168] Started disk
2023-02-15 00:11:01,158 INFO    SystemMonitor:3648 [interfaces.py:start():168] Started gpu
2023-02-15 00:11:01,158 INFO    SystemMonitor:3648 [interfaces.py:start():168] Started memory
2023-02-15 00:11:01,159 INFO    SystemMonitor:3648 [interfaces.py:start():168] Started network
2023-02-15 00:11:01,168 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:01,169 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:01,227 DEBUG   HandlerThread:3648 [system_info.py:probe():195] Probing system
2023-02-15 00:11:01,249 DEBUG   HandlerThread:3648 [system_info.py:_probe_git():180] Probing git
2023-02-15 00:11:01,302 DEBUG   HandlerThread:3648 [system_info.py:_probe_git():188] Probing git done
2023-02-15 00:11:01,303 DEBUG   HandlerThread:3648 [system_info.py:probe():241] Probing system done
2023-02-15 00:11:01,303 DEBUG   HandlerThread:3648 [system_monitor.py:probe():181] {'os': 'Linux-5.15.0-60-generic-x86_64-with-debian-bullseye-sid', 'python': '3.7.16', 'heartbeatAt': '2023-02-14T15:11:01.227848', 'startedAt': '2023-02-14T15:11:00.405523', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'train.py', 'codePath': 'sac/train.py', 'git': {'remote': 'git@github.com:filyps98/JacoRLDR.git', 'commit': '63fbaf70651882592c2f5bb870d93a37351552ef'}, 'email': 'pippo.ronco@gmail.com', 'root': '/home/filippo/Desktop/Project/Experiments/JacoRLDR', 'host': 'filippo-X510UQ', 'username': 'filippo', 'executable': '/home/filippo/miniconda3/envs/robot/bin/python', 'cpu_count': 2, 'cpu_count_logical': 4, 'cpu_freq': {'current': 2472.4815, 'min': 400.0, 'max': 3500.0}, 'cpu_freq_per_core': [{'current': 2900.0, 'min': 400.0, 'max': 3500.0}, {'current': 900.008, 'min': 400.0, 'max': 3500.0}, {'current': 900.047, 'min': 400.0, 'max': 3500.0}, {'current': 2900.0, 'min': 400.0, 'max': 3500.0}], 'disk': {'total': 96.18362426757812, 'used': 46.52611541748047}, 'gpu': 'NVIDIA GeForce 940MX', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce 940MX', 'memory_total': 2147483648}], 'memory': {'total': 7.649139404296875}}
2023-02-15 00:11:01,304 INFO    HandlerThread:3648 [system_monitor.py:probe():182] Finished collecting system info
2023-02-15 00:11:01,304 INFO    HandlerThread:3648 [system_monitor.py:probe():185] Publishing system info
2023-02-15 00:11:01,304 DEBUG   HandlerThread:3648 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2023-02-15 00:11:01,305 DEBUG   HandlerThread:3648 [system_info.py:_save_pip():67] Saving pip packages done
2023-02-15 00:11:01,305 DEBUG   HandlerThread:3648 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2023-02-15 00:11:02,069 INFO    Thread-13 :3648 [dir_watcher.py:_on_file_created():275] file/dir created: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/files/wandb-summary.json
2023-02-15 00:11:02,070 INFO    Thread-13 :3648 [dir_watcher.py:_on_file_created():275] file/dir created: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/files/requirements.txt
2023-02-15 00:11:02,070 INFO    Thread-13 :3648 [dir_watcher.py:_on_file_created():275] file/dir created: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/files/conda-environment.yaml
2023-02-15 00:11:03,170 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:03,170 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:03,607 DEBUG   HandlerThread:3648 [system_info.py:_save_conda():86] Saving conda packages done
2023-02-15 00:11:03,607 INFO    HandlerThread:3648 [system_monitor.py:probe():187] Finished publishing system info
2023-02-15 00:11:03,614 DEBUG   SenderThread:3648 [sender.py:send():334] send: files
2023-02-15 00:11:03,615 INFO    SenderThread:3648 [sender.py:_save_file():1319] saving file wandb-metadata.json with policy now
2023-02-15 00:11:03,645 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: stop_status
2023-02-15 00:11:03,646 DEBUG   SenderThread:3648 [sender.py:send_request():361] send_request: stop_status
2023-02-15 00:11:04,068 INFO    Thread-13 :3648 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/files/conda-environment.yaml
2023-02-15 00:11:04,070 INFO    Thread-13 :3648 [dir_watcher.py:_on_file_created():275] file/dir created: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/files/wandb-metadata.json
2023-02-15 00:11:04,107 DEBUG   SenderThread:3648 [sender.py:send():334] send: telemetry
2023-02-15 00:11:04,721 INFO    Thread-16 :3648 [upload_job.py:push():143] Uploaded file /tmp/tmp8oaio228wandb/6javpinq-wandb-metadata.json
2023-02-15 00:11:05,069 INFO    Thread-13 :3648 [dir_watcher.py:_on_file_created():275] file/dir created: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/files/output.log
2023-02-15 00:11:05,172 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:05,172 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:06,110 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: status_report
2023-02-15 00:11:07,071 INFO    Thread-13 :3648 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/filippo/Desktop/Project/Experiments/JacoRLDR/sac/wandb/run-20230215_001100-untbe0sx/files/output.log
2023-02-15 00:11:07,174 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:07,175 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:09,176 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:09,177 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:11,111 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: status_report
2023-02-15 00:11:11,178 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:11,178 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:13,179 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:13,179 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:15,180 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:15,181 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:16,113 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: status_report
2023-02-15 00:11:17,182 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:17,183 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:19,184 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:19,184 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:21,114 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: status_report
2023-02-15 00:11:21,184 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:21,185 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:23,186 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:23,186 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:25,187 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:25,187 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:26,114 DEBUG   HandlerThread:3648 [handler.py:handle_request():144] handle_request: status_report
2023-02-15 00:11:27,180 ERROR   cpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=3629)
2023-02-15 00:11:27,180 ERROR   cpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=3629)
2023-02-15 00:11:27,181 ERROR   memory    :3648 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=3629)
2023-02-15 00:11:27,181 ERROR   memory    :3648 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=3629)
2023-02-15 00:11:27,187 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:27,187 ERROR   gpu       :3648 [interfaces.py:monitor():129] Failed to sample metric: Not Supported
2023-02-15 00:11:29,155 INFO    MainThread:3648 [internal.py:handle_exit():77] Internal process exited
